examples/forkawesome_icons.py

```diff
@@ -0,0 +1,804 @@
+# SPDX-FileCopyrightText: 2020 Jeff Epler
+# SPDX-License-Identifier: MIT
```

Comment:
@jepler I got this file from your blog post, but I didn't see a license. I assumed MIT, but let me know if this should be something different, or if this helper file did not originate with you.

Reply:
that's fine, thanks